vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditReferenceController.java

```diff
@@ -0,0 +1,69 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.asu.diging.vspace.core.model.IReference;
+import edu.asu.diging.vspace.core.model.impl.Reference;
+import edu.asu.diging.vspace.core.services.IReferenceManager;
+
+@Controller
+public class EditReferenceController {
+
+    @Autowired
+    private IReferenceManager referenceManager;
+
+    @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/bibliography/{biblioId}/reference/edit", method = RequestMethod.POST)
```

Comment:
should be consistent, either
`.../reference/edit` and `.../reference/add`
or
`.../references/edit` and `.../references/add`